content/shell/android/shell_apk/src/org/chromium/content_shell_apk/ContentShellActivity.java

```diff
@@ -73,10 +73,8 @@ protected void onCreate(final Bundle savedInstanceState) {
         mWindowAndroid.setAnimationPlaceholderView(
                 mShellManager.getContentViewRenderView().getSurfaceView());
 
-        // TODO(cobalt, b/376148547): set Chrobalt initial url and remove this function.
-        if (mStartupUrl.isEmpty()) {
-            mStartupUrl = getUrlFromIntent(getIntent());
-        }
+        mStartupUrl = getUrlFromIntent(getIntent());
+
```

Comment:
nit: it should be no extra line from the original code, we should keep it as it was

Reply:
oh shoot I was looking at the upstream code: https://source.chromium.org/chromium/chromium/src/+/main:content/shell/android/shell_apk/src/org/chromium/content_shell_apk/ContentShellActivity.java;drc=62a63ab0b2ee9082b1bb25a74187bced2f599858;l=79

changing now.